changelogs/2023-01-09-sftpgo-get-key.md

```diff
@@ -0,0 +1,4 @@
+### Added
+
+- `sftpgo/get_admin_api_key.sh` now has an optional flag to force it to
+  overwrite an existing key in `settings`.
```

Comment:
Should probably state here that the flag is `--force` explicitly so people don't have to look at the script code in addition to reading this changelog entry.

Reply:
That's in the docs. I don't generally put detailed info in the changelogs.

Reply:
Ah, that's true. But if it were me I'd note it in both for convenience of those reading the changelog. :stuck_out_tongue_winking_eye: